themes/digital.gov/layouts/partials/core/home/news_featured.html

```diff
@@ -24,8 +24,8 @@ <h2>News and Events</h2>
         {{/* Future Events =========================== */}}
         {{/* gets the # of news items to display from the config.yml */}}
         {{- $events_count := .Site.Params.events_count -}}
-        {{- $events := where .Site.RegularPages "Section" "events" -}}
-        {{- $events := $events | first $events_count }}
+        {{- $events_future := where (where .Site.RegularPages "Section" "events") ".Date.Unix" ">" now.Unix -}}
+        {{- $events_future := $events_future | first $events_count }}
```

Comment:
Refactored `$events` to `$future_events` and updated logic to only select events that are greater than current day to make check more explicit and avoid overlapping with past events.